executor/linux/secret.go

```diff
@@ -108,40 +109,44 @@ func (s *secretSvc) destroy(ctx context.Context, ctn *pipeline.Container) error
 	logger := s.client.logger.WithField("secret", ctn.Name)
 
 	// load the step from the client
-	step, err := s.client.loadStep(s.client.init.ID)
+	secret, err := step.Load(s.client.init, &s.client.steps)
 	if err != nil {
 		// create the step from the container
-		step = new(library.Step)
-		step.SetName(ctn.Name)
-		step.SetNumber(ctn.Number)
-		step.SetStatus(constants.StatusPending)
-		step.SetHost(ctn.Environment["VELA_HOST"])
-		step.SetRuntime(ctn.Environment["VELA_RUNTIME"])
-		step.SetDistribution(ctn.Environment["VELA_DISTRIBUTION"])
+		secret = new(library.Step)
```

Comment:
should this be `s`? Doesn't seem like a secret

Reply:
The reason it isn't `s` is due to the function header where `s` is already taken:

https://github.com/go-vela/pkg-executor/blob/802c16e3d11aa696727437394ca8e0b7998c605c/executor/linux/secret.go#L105

In regards to not seeming like a secret, I would agree based off the code 👍 

But, this code is responsible for executing the containers for external secrets so it is kind of a gray area.

Reply:
👍 it just looked different than similar sections of code!